api-docs/api/fuse/triggers/actions.json

```diff
@@ -786,7 +786,7 @@
           {
             "comment": {
               "brief": "Toggles the state of a toggleable component.",
-              "full": "Toggles the state of a toggleable component.\n\nIt is not reccommended to use this trigger action to toggle a logical state in your app. For that, use\nan Observable boolean in JavaScript and manipulate its value in a callback.\n\n## Example\n\n    <StackPanel>\n        <Switch ux:Name=\"switch1\" />\n\n        <Button Text=\"Toggle!\">\n            <Clicked>\n                <Toggle Target=\"switch1\" />\n            </Clicked>\n        </Button>\n    </StackPanel>"
+              "full": "Toggles the state of a toggleable component.\n\nIt is not recommended to use this trigger action to toggle a logical state in your app. For that, use\nan Observable boolean in JavaScript and manipulate its value in a callback.\n\n## Example\n\n    <StackPanel>\n        <Switch ux:Name=\"switch1\" />\n\n        <Button Text=\"Toggle!\">\n            <Clicked>\n                <Toggle Target=\"switch1\" />\n            </Clicked>\n        </Button>\n    </StackPanel>"
```

Comment:
This comes from the fuselibs sources, and should be fixed there instead:

https://github.com/fuse-open/fuselibs/search?q=reccommended&unscoped_q=reccommended

Reply:
The same applies to a few other places as well. I think it should be obvious which ones.